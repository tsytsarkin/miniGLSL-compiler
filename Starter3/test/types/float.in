{
    float i;
    float j = float(1.34234);
    j = 123.0;
    i = j;
}
