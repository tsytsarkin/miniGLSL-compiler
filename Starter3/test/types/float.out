  1: {
  2:     float i;
  3:     float j = 1.34234;
  4:     j = 123.0;
  5: }
 (SCOPE (DECLARATIONS (DECLARATION i float) (DECLARATION j float 1.342340)) (STATEMENTS (ASSIGN float j 123.000000)))
